### PR TITLE
Support SPM (Swift Package Manager)

### PR DESCRIPTION
**Pull request by tung-vu-td**
This PR will:
- Add support for SPM
- Install Cocoapod will now need to add `use_framework!` to the beginning of Podfile
- Remove support for framework

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -1,4 +1,5 @@
-platform :ios, '7.0'
+platform :ios, '12.0'
+use_frameworks!
 
 target 'TreasureData' do
   pod 'KeenClientTD', '= 3.3.0'
```
Comment (kietdo360) [inline Podfile:5]: still 3.3 ?

**Review by kietdo360 — APPROVED** [review]
LGTM with minor comment

File: Podfile (hunk 1 above)
Comment (tung-vu-td) [reply Podfile]: Thanks for catching this 🙇

**Review by nmpennypacker — APPROVED** [review]
Reviewed for data exfiltrations

**Pull request merged**